src/OneExplorer/OneExplorer.ts

```diff
@@ -163,6 +163,50 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     this._onDidChangeTreeData.fire();
   }
 
+  // TODO: Add move()
+
+  /**
+   * Rename a file of all types of nodes (baseModel, derivedModel, config) excepts for directory.
+   * It only alters the file name, not the path.
+   */
+  rename(oneNode: OneNode): void {
+    // TODO: prohibit special characters for security ('..', '*', etc)
+    let warningMessage;
+    if (oneNode.node.type === NodeType.baseModel) {
+      // TODO automatically change the corresponding files
+      warningMessage = 'WARNING: You may need to change input paths in the following cfg files:\n';
+      for (const conf of oneNode.node.childNodes) {
+        warningMessage += `${conf.name} \n`;
+      }
+    } else {
+      warningMessage = 'WARNING: Renaming may result in some unexpected changes on the tree view.';
+    }
+
+    vscode.window
+        .showInputBox({title: warningMessage, placeHolder: 'Enter(proceed) / Escape(cancel)'})
```

Comment:
You don't need to put `Enter(proceed) / Escape(cancel)` string. Because ShowInputBox already provides the help message on bottom.
![rename_help_message](https://user-images.githubusercontent.com/7223627/170408010-0c30f19c-aa03-4fea-9ba0-353d06511d20.png)

Reply:
Sure. Let me remove this then.